app/components/PageWrapper.js

```diff
@@ -29,11 +29,13 @@ class PageWrapper extends Component {
   componentDidMount() {
     ipcRenderer.on('update-downloaded', this.onAppUpgrade);
     ipcRenderer.on('play-replay', this.onPlayReplay);
+    ipcRenderer.on('play-local-replay', this.onPlayLocalReplay);
   }
 
   componentWillUnmount() {
     ipcRenderer.removeListener('update-downloaded', this.onAppUpgrade);
     ipcRenderer.removeListener('play-replay', this.onPlayReplay);
+    ipcRenderer.removeListener('play-local-replay', this.onPlayLocalReplay);
```

Comment:
No longer an anonymous function because I realized that the listener passes variables